src/Symfony/Component/Messenger/Command/MessengerDebugCommand.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * A console command to debug Messenger information.
+ *
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ *
+ * @experimental in 4.1
+ */
+class MessengerDebugCommand extends Command
```

Comment:
Shouldn't it be named `DebugCommand` as debug commands in other components?

Reply:
Good point @yceruto, you are right. Commands within components' namespace are not prefixed with the component name, let's keep it this way :)

Reply:
Renamed 👍